Description: remove failing tests that prevent installation
 The testing script references a non-existing video.mp4 that causes the script
 to fail, which prevents cmake to install the files to their target location.
Author: Roland Hieber <rohieb@rohieb.name>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,9 +23,6 @@
 configure_file(res/nordlicht.pc.in res/nordlicht.pc @ONLY)
 configure_file(src/version.h.in src/version.h @ONLY)
 
-enable_testing()
-add_test(test-bin ${CMAKE_SOURCE_DIR}/test/test-bin.sh)
-
 install(TARGETS nordlicht DESTINATION ${LIB_INSTALL_DIR})
 install(TARGETS nordlicht-bin DESTINATION bin)
 install(FILES src/nordlicht.h DESTINATION include)
